cfglib.py

```diff
@@ -200,17 +200,17 @@
                 help='live-migration without shared_storage'),
     cfg.StrOpt('host_eph_drv', default='-',
                help='host ephemeral drive'),
-    cfg.StrOpt('connection', default='mysql+mysqlconnector',
+    cfg.StrOpt('db_connection', default='mysql+mysqlconnector',
                help='driver for db connection'),
-    cfg.StrOpt('host', default='',
+    cfg.StrOpt('db_host', default='',
                help='compute mysql node ip address'),
-    cfg.IntOpt('port', default='3306',
+    cfg.IntOpt('db_port', default='3306',
                help='port for mysql connection'),
     cfg.StrOpt('database_name', default='',
```

Comment:
Maybe It is better to change this option name too for uniformity? (to `db_name`). 
The same for all option groups.